core/src/settings.ts

```diff
@@ -25,6 +25,7 @@ export const LISTENER_DELAY = Number(process.env.LISTENER_DELAY) || 500
 export const VRF_FULFILL_GAS_MINIMUM = 450_000
```

Comment:
How about reverting back to 400k?

Reply:
Yes. Let's make VRF_FULFILL_GAS_MINIMUM = 400k.